src/hotspot/share/cds/filemap.hpp

```diff
@@ -180,9 +180,10 @@ class FileMapRegion: private CDSFileMapRegion {
 };
 
 class FileMapHeader: private CDSFileMapHeaderBase {
-  friend class CDSOffsets;
   friend class VMStructs;
+  friend class CDSConstants;
```

Comment:
It's better to sort the classes alphabetically. CDSConstants should come before VMStructs.